src/mediapipe_internal/pythonnoderesource.cpp

```diff
@@ -83,6 +107,9 @@ Status PythonNodeResource::createPythonNodeResource(std::shared_ptr<PythonNodeRe
         py::object OvmsPythonModel = script.attr("OvmsPythonModel");
         py::object pythonModel = OvmsPythonModel();
         py::object kwargsParam = pybind11::dict();
+        kwargsParam["node_name"] = node_name;
+        kwargsParam["input_stream"] = inputStream;
```

Comment:
As far I understand we want to inform the script about input/output names it will receive.
Why do we need to concatenate single string, if we could create py::list?
It would be more convenient for user to access kwargs["input_streams"] as list